src/Open3D/GUI/SceneWidget.h

```diff
@@ -54,6 +54,7 @@ class SceneWidget : public Widget {
 
     void SetBackgroundColor(const Color& color);
     void SetDiscardBuffers(const visualization::View::TargetBuffers& buffers);
+    void SetViewMode(visualization::View::Mode mode);
```

Comment:
It seems like open3d::gui ought to abstract away open3d::visualization where possible. Unfortunately, this would mean duplicating the enum in open3d::gui::SceneWidget, which seems like a bit of a waste. I don't have a strong opinion here, but as a caller, I'm always a little frustrated when a function uses something I have to go look up in a completely different section of the code (even if it is just Cmd-RightClick in Xcode)

Reply:
I see there two possible solutions:
1. I'll add another enum with modes to `SceneWidget`
2. I'll remove `SetDiscardBuffers` and `SetViewMode` methods and add `GetView` method.
I personally prefer second because both methods more about rendering than about widget. And we already has this `GetScene` method also.

Reply:
Either are fine with me, although I agree that #2 does sound like the better option.